integration/simulation/l1_network.go

```diff
@@ -52,13 +63,14 @@ func (n *L1NetworkCfg) BroadcastTx(tx common2.EncodedL1Tx) {
 	}
 }
 
-func (n *L1NetworkCfg) Start(delay time.Duration) {
+// Start kicks off the l1 nodes waiting 1 second between each node
+func (n *L1NetworkCfg) Start() {
 	// Start l1 nodes
 	for _, m := range n.nodes {
 		t := m
 		go t.Start()
 		// don't start everything at once
-		time.Sleep(delay)
+		time.Sleep(time.Duration(NODE_BOOTUP_DELAY_MS * 1_000))
```

Comment:
After you figured out where the delay of processing came from (thanks btw 😄 ), I changed that node bootup delay on purpose. 

In my head using the same duration in two separate places, it's not great. Like if we're changing the`avgBlockDuration` because we want to see what different params do, we don't necessarily expect it to change the node delay time.

Reply:
well, the point is to not start all nodes at once. You'd start the as blocks get created. 
So basically you want to start them as blocks progress

Reply:
Was not seeing that effect 👍  thanks!